ci/azure-jobs-doxygen.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- job: Doxygen
+  pool:
+    vmImage: 'ubuntu-16.04'
+  container: librepcb/librepcb-dev:ubuntu-14.04-1
```

Comment:
Why Ubuntu 14.04? It is end of life since April 2019.

Reply:
I tried a more recent Ubuntu version, but for some reason Doxygen failed to run `dot` to create the diagrams. As 14.04 was already used until now, I kept this for the moment - but yes, this should be fixed some day ;)

Reply:
Alright. I created https://github.com/LibrePCB/LibrePCB/issues/461 to resolve it later.

Reply:
Thanks! :+1: